File: src/main/java/com/muxiu1997/sharewhereiam/loader/KeyLoader.java
package com.muxiu1997.sharewhereiam.loader;

import com.muxiu1997.sharewhereiam.client.key.KeyShare;
import cpw.mods.fml.client.registry.ClientRegistry;
import cpw.mods.fml.relauncher.Side;
import cpw.mods.fml.relauncher.SideOnly;

@SideOnly(Side.CLIENT)
public class KeyLoader {

    public static final KeyLoader INSTANCE = new KeyLoader();

    public void clientLoad() {
        ClientRegistry.registerKeyBinding(KeyShare.INSTANCE);
    }
}

File: src/main/kotlin/com/muxiu1997/sharewhereiam/loader/KeyLoader.kt
package com.muxiu1997.sharewhereiam.loader

import com.muxiu1997.sharewhereiam.client.KeyShare
import cpw.mods.fml.client.registry.ClientRegistry
import cpw.mods.fml.relauncher.Side
import cpw.mods.fml.relauncher.SideOnly

@SideOnly(Side.CLIENT)
object KeyLoader {
    fun clientLoad() {
        ClientRegistry.registerKeyBinding(KeyShare)
    }
}
